src/components/cc-overview/cc-overview.stories.js

```diff
@@ -28,6 +28,10 @@ const conf = {
     cc-overview {
       min-height: 750px;
     }
+    
+    cc-tile-metrics {
+      max-width: 100% !important;
+    }
```

Comment:
Curious to know why we need this

Reply:
We need that because we have a min-width on the `tile-metrics` stories and if we don't do that the tile doesn't scale in width and isn't responsive